src/controllers/user.py

```diff
@@ -22,14 +22,17 @@ def create_user(body):
     return result, code
 
 
-def get_all_users():
+def get_all_users(filter=None):
+    if filter:
+        result, code = db.get_all(User, {"invalid": 5})
+        return result, code
```

Comment:
If we don't have any filter functionality for user, this need not be done. There is no need to test functionality that does not exist